### DISCUSSION TITLE
Not receiving my XCH in the official wallet

### DISCUSSION THREAD

**Discussion by gaspetrou** [Chia-Network/chia-blockchain#15487] [Support]
Hello,
I bought via Kucoin XCH and also I have creat an official CHIA Wallet at my pc running win11 pro.

I proceed with 2 transactions to test my official CHIA Wallet
1st transfering 0,1 XCH to my CHIA official wallet xch1gqj5zpudm0eywrq4pgq5ee2jck3hd4luz2s6m5wg3lv8c5pp7gdqq6dvzk
TXHash 
0x94f2c94f764dc21d0b7bb490b6e9eb433785c0f3410dbe029831077351547b05

and 2nd transfer 1,1 XCH to the same no official CHIA wallet with 
TXHash
0x4d71f4c098a3d09b69e9f539bca57692443bb64265bdc2c4e79967977848340c

At my CHIA Wallet app the summary balance is 0 XCH !!!

Also I have received confirmation from Kucoin for the succesfuly transfer and at CHIA Scan net I can see that the transfer is done but in my wallet the balance is 0.
Also my derivation index is 428.

Awaiting for your help many thanks in advance.

**Comment by gaspetrou** [comment]
Furthermore at my CHIA address xch1gqj5zpudm0eywrq4pgq5ee2jck3hd4luz2s6m5wg3lv8c5pp7gdqq6dvzk I can see my both credited XCH and everything is done !

**Comment by gaspetrou** [comment]
is this project dead????

**Comment by gaspetrou** [comment]
the support is working or not?

**Comment by sasani65** [comment]
did you recive your xchia?

**Comment by BrandtH22** [comment]
Good afternoon gaspetrou, I am sorry for the delayed response.
The majority of our support is currently focused on the discord server (https://discord.gg/chia) but we will recognize that additional attention needs to be given to the github support issues and we are working to improve on this.

Have you received the funds to your xch address? (one of your previous messages seem to suggest so but I want to be sure)

**Comment by sasani65** [comment]
Hi..I didn’t receive my chia yet..how can I find my funds?

**Reply by BrandtH22** [reply]
Please feel free to jump into the support channel on our discord server (https://discord.gg/chia).

We will need more information on what you are experiencing though:
Do you have any transaction information for the xch you were supposed to receive?
From where were the funds being sent?

**Comment by sasani65** [comment]
On Fri, Jul 28, 2023 at 19:53 ClydeWallace22 ***@***.***>
wrote:

> Please feel free to jump into the support channel on our discord server (
> https://discord.gg/chia).
>
> We will need more information on what you are experiencing though:
> Do you have any transaction information for the xch you were supposed to
> receive?
> From where were the funds being sent?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/15487#discussioncomment-6578853>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BBE5IIHGVI3ABUQ7NZV47UTXSQDEFANCNFSM6AAAAAAZCUOLHA>
> .
> You are receiving this because you commented.Message ID:
> <Chia-Network/chia-blockchain/repo-discussions/15487/comments/6578853@
> github.com>
>
I sent you …the screenshot of my transaction from kucoin to your native
chia wallet

**Reply by BrandtH22** [reply]
I, unfortunately, have not seen the screenshot in this thread but appreciate you providing the information as to from where the funds were sent.

Please feel free to jump into the support channel on our discord server (https://discord.gg/chia) for more immediate and thorough support.

Generally users who experience these types of issues will:
1. Verify on an explorer that the funds were sent to the address provided to the exchange and is in an "unspent" state in that address
2. If the above is correct and funds are not found in a wallet that controls that xch address, then the sers will use the gui option to resync their wallet (settings / advanced / Resync) 
3. If that does not resolve the issues then users will generally close the chia client, delete the wallet db, then resync the wallet from scratch
4. If this still does not resolve the issue then users will connect their wallet to a local trusted full node and perform a resync from there (trusted full nodes require fewer validations than general networked full nodes enabling a faster and more accurate sync)

**Comment by sasani65** [comment]
Hi,reinstall my chia wallet and resync it but i cant see my funds yet,how

**Reply by BrandtH22** [reply]
Hey @sasani65 , if you can provide more information regarding the xch address to which the funds were sent that will help us determine what troubleshooting can be conducted to identify where the funds are currently.

Please feel free to jump into the support channel on our discord server (https://discord.gg/chia) for more immediate and thorough support.

Generally users who experience these types of issues will:

1. Verify on an explorer that the funds were sent to the address provided to the exchange and is in an "unspent" state in that address
2. If the above is correct and funds are not found in a wallet that controls that xch address, then the sers will use the gui option to resync their wallet (settings / advanced / Resync)
3. If that does not resolve the issues then users will generally close the chia client, delete the wallet db, then resync the wallet from scratch
4. If this still does not resolve the issue then users will connect their wallet to a local trusted full node and perform a resync from there (trusted full nodes require fewer validations than general networked full nodes enabling a faster and more accurate sync)

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
